etcdserver/etcdhttp/client.go

```diff
@@ -98,6 +100,7 @@ func NewClientHandler(server *etcdserver.EtcdServer) http.Handler {
 	mux.HandleFunc(statsPrefix+"/self", sh.serveSelf)
 	mux.HandleFunc(statsPrefix+"/leader", sh.serveLeader)
 	mux.HandleFunc(varsPath, serveVars)
+	mux.HandleFunc(configPath+"/local/debug", debugHandleFunc)
```

Comment:
maybe `/local/log`? it uses only true and false for [-debug flag](https://github.com/coreos/etcd/blob/master/Documentation/configuration.md#logging-flags).